criu/kerndat.c

```diff
@@ -986,6 +987,40 @@ static int kerndat_tun_netns(void)
 	return check_tun_netns_cr(&kdat.tun_ns);
 }
 
+static bool kerndat_has_clone3_set_tid(void)
+{
+	pid_t pid;
+	struct _clone_args args = {};
+
+#ifndef CONFIG_X86_64
+	kdat.has_clone3_set_tid = false;
```

Comment:
is clone3_set_tid supported by other architectures?

Reply:
It is supported on all CRIU architectures, but it needs an assembler wrapper which I currently only have implemented for x86_64. Once I have the assembler wrappers for all architectures, this will be removed.

Reply:
adding a comment to make it easier to understand